hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleInjectionModule.java

```diff
@@ -19,9 +19,13 @@
 import dagger.Binds;
 import dagger.Module;
 import javax.inject.Singleton;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 @Module
 public interface ThrottleInjectionModule {
+    Logger log = LogManager.getLogger(ThrottleInjectionModule.class);
```

Comment:
Is this used anywhere?

Reply:
I had some temporary implementation at some point using it but currently it's not used, will remove it.